CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ More than 200 awesome developers have contributed to the `gradio` library, and w
 
 Prerequisites:
 
-- [Python 3.8+](https://www.python.org/downloads/)
```

Comment:
3.8 is supported though?

Reply:
yep ! but python is now modified..
if you want , you can merge this pr..
otherwise you can close, depends upon your perception

Reply:
What do you mean by `but python is now modified..` ?

Reply:
i mean python is updated to version.

Reply:
Oh , okay 👍